packages/common/src/api/tan-query/useWalletCollectibles.ts

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { useAudiusQueryContext } from '~/audius-query/AudiusQueryContext'
+import { Chain } from '~/models'
+
+import { QUERY_KEYS } from './queryKeys'
+import { QueryOptions } from './types'
+
+type UseWalletCollectibles = {
+  /** Ethereum or Solana wallet address */
+  address: string
+  chain: Chain
+}
+
+export const getCollectiblesQueryKey = ({
+  address,
+  chain
+}: UseWalletCollectibles) => [QUERY_KEYS.walletCollectibles, chain, address]
+
+/**
+ * Query function for getting the collectibles of an Ethereum or Solana wallet.
+ */
+export const useWalletCollectibles = (
```

Comment:
CC: @schottra I wonder if there ought to be a crossover between this and `useUserCollectibles`? Maybe what I did here is the wrong strategy...